cms/templates/settings.html

```diff
@@ -364,10 +364,72 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                       <input type="text" dir="ltr" class="long new-course-image-url" id="course-image-url" value="" placeholder="${_("Your course image URL")}" autocomplete="off" />
                       <span class="tip tip-stacked">${_("Please provide a valid path and name to your course image (Note: only JPEG or PNG format supported)")}</span>
                     </div>
-                    <button type="button" class="action action-upload-image">${_("Upload Course Image")}</button>
+                    <button type="button" class="action action-upload-image" id="upload-course-image">${_("Upload Course Image")}</button>
                   </div>
                 </li>
 
+                % if enable_extended_course_details:
+                <li class="field image" id="field-hero-image">
+                  <label for="hero-image-url">${_("Course Details Background")}</label>
```

Comment:
Nit: Given the distance between the label and the input field associated with it, described_by might be a better choice than "for" for linking the elements. However, since this is the same pattern used in the course image HTML, I'm OK with leaving it as-is.